src/fasta.zig

```diff
@@ -30,8 +30,8 @@ fn repeatAndWrap(out: *OutStream(File.WriteError), comptime sequence: []const u8
         const rem = count - idx;
         const line_length = std.math.min(usize(max_line_length), rem);
 
-        _ = out.write(padded_sequence[off .. off + line_length]);
-        _ = out.writeByte('\n');
+        _ = out.write(padded_sequence[off .. off + line_length]) catch {};
```

Comment:
There are a bunch of instances of `catch {}` now that error checking is mandatory. Maybe this should be `catch unreachable`, but for this pull request, I matched previous semantics.

Reply:
Yeah, can't remember why this was done like this. I'll update in another commit.